feature-libs/user/profile/components/update-password/update-password-component.service.spec.ts

```diff
@@ -166,14 +169,41 @@ describe('UpdatePasswordComponentService', () => {
     });
 
     describe('error', () => {
-      it('should not update the password', () => {
+      it('should not update the password when old password is empty', () => {
         newPassword.setValue('testpassword123');
         service.updatePassword();
         expect(userPasswordFacade.update).not.toHaveBeenCalled();
         expect(globalMessageService.add).not.toHaveBeenCalled();
         expect(routingService.go).not.toHaveBeenCalled();
         expect(authService.coreLogout).not.toHaveBeenCalled();
       });
+      it('should get access denied error message text with agent modifying customer password when feature level is above 6.3', () => {
+        spyOn(featureConfig, 'isLevel').and.returnValue(true);
+        (userPasswordFacade.update as jasmine.Spy).and.returnValue(
+          throwError({ details: [{ type: 'AccessDeniedError' }] })
+        );
+
+        oldPassword.setValue('Old123!');
+        newPassword.setValue('New123!');
+        newPasswordConfirm.setValue('New123!');
+        service.updatePassword();
+        expect(globalMessageService.add).toHaveBeenCalledWith(
+          { key: 'updatePasswordForm.accessDeniedError' },
+          GlobalMessageType.MSG_TYPE_ERROR
+        );
+      });
+      it('should not get access denied error message text with agent modifying customer password when feature level is below 6.3', () => {
```

Comment:
I think adding a version description is unnecessary, could you please delete it? `when feature level is above 6.3 `